src/com/moclojer/server.clj

```diff
@@ -64,7 +116,8 @@
         http-port (or (some-> (System/getenv "PORT")
                               Integer/parseInt)
                       8000)
-        http-start (if start? http/start ::http/service-fn)]
+        http-start (if start? http/start ::http/service-fn)
+        swagger? (or (System/getenv "SWAGGER") false)]
```

Comment:
this will make the retro compatibility works with pedestal and tests will not break

Reply:
I would leave it `true` by default, to have Swagger by default.